tooling/Microsoft.VisualStudio.RazorExtension/Microsoft.VisualStudio.RazorExtension.csproj

```diff
@@ -45,7 +45,7 @@
     <IncludeDebugSymbolsInVSIXContainer>true</IncludeDebugSymbolsInVSIXContainer>
```

Comment:
False?

Reply:
Pfff you saw nothing; totally didn't just force push that away... I promise